editor/plugins/script_text_editor.cpp

```diff
@@ -2158,8 +2158,8 @@ void ScriptTextEditor::register_editor() {
 	// Leave these at zero, same can be accomplished with tab/shift-tab, including selection.
 	// The next/previous in history shortcut in this case makes a lot more sense.
 
-	ED_SHORTCUT("script_text_editor/indent_left", TTR("Indent Left"), Key::NONE);
-	ED_SHORTCUT("script_text_editor/indent_right", TTR("Indent Right"), Key::NONE);
+	ED_SHORTCUT("script_text_editor/dedent", TTR("Dedent"), KeyModifierMask::SHIFT | Key::TAB);
+	ED_SHORTCUT("script_text_editor/indent", TTR("Indent"), Key::NONE);
```

Comment:
I think "indent" should come first (here and in the menus).
It made sense to have left before right (at least for LTR centric users), but not to have dedent before indent.